detections/endpoint/windows_obfuscated_files_or_information_via_rar_sfx.yml

```diff
@@ -0,0 +1,68 @@
+name: Windows Obfuscated Files or Information via RAR SFX
+id: 4ab6862b-ce88-4223-96c0-f6da2cffb898
+version: 1
+date: '2024-12-12'
+author: Teoderick Contreras, Splunk
+data_sources:
+- Sysmon Event ID 11
+type: TTP
```

Comment:
From VT data we can see that the creation of such file is pretty common
![image](https://github.com/user-attachments/assets/a36aaedb-67b0-406a-9141-94491798fc33)


Hence putting it as Hunting is wise.
```suggestion
type: Hunting
```

Reply:
noted!